plugins/@grouparoo/pipedrive/src/lib/client.ts

```diff
@@ -0,0 +1,112 @@
+import axios, { AxiosInstance } from "axios";
+
+export class PipedriveClient {
+  request: AxiosInstance;
+
+  constructor(apiToken: string) {
+    this.request = axios.create({
+      baseURL: "https://api.pipedrive.com/v1",
+      params: {
+        api_token: apiToken,
+      },
+    });
+  }
+
+  async getCurrentUser() {
+    const { data } = await this.request.get("/users/me");
+    return data.data;
+  }
+
+  async findPersonIdByEmail(email: string): Promise<number | null> {
```

Comment:
Not a blocker, FYI for later - I don't think you ever need to ` | null` in typescript.  Null (or undefined) are always valid return types, e.g.: 

<img width="547" alt="Screen Shot 2021-03-22 at 6 24 35 PM" src="https://user-images.githubusercontent.com/303226/112078546-de123a80-8b3b-11eb-9b2c-169b85577f95.png">